src/main/java/de/siphalor/amecs/impl/mixin/MixinInputUtilType.java

```diff
@@ -1,21 +1,23 @@
 package de.siphalor.amecs.impl.mixin;
 
-import de.siphalor.amecs.api.KeyBindingUtils;
-import net.fabricmc.api.EnvType;
-import net.fabricmc.api.Environment;
-import net.minecraft.client.util.InputUtil;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
+import de.siphalor.amecs.api.KeyBindingUtils;
+import de.siphalor.amecs.impl.AmecsAPI;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.client.util.InputUtil;
+
 @Mixin(InputUtil.Type.class)
 @Environment(EnvType.CLIENT)
 public abstract class MixinInputUtilType {
 	@SuppressWarnings("UnresolvedMixinReference")
 	@Inject(method = "<clinit>", at = @At("RETURN"))
 	private static void onRegisterKeyCodes(CallbackInfo callbackInfo) {
-		InputUtil.Type.mapKey(InputUtil.Type.MOUSE, "amecsapi.key.mouse.scroll.up", KeyBindingUtils.MOUSE_SCROLL_UP);
-		InputUtil.Type.mapKey(InputUtil.Type.MOUSE, "amecsapi.key.mouse.scroll.down", KeyBindingUtils.MOUSE_SCROLL_DOWN);
+		InputUtil.Type.mapKey(InputUtil.Type.MOUSE, AmecsAPI.makeKeyID("mouse.scroll.up"), KeyBindingUtils.MOUSE_SCROLL_UP);
+		InputUtil.Type.mapKey(InputUtil.Type.MOUSE, AmecsAPI.makeKeyID("mouse.scroll.down"), KeyBindingUtils.MOUSE_SCROLL_DOWN);
```

Comment:
> Don't you use the same method for giving ids to key bindings?
Yes I do.

> The swapping of the ordering of the namespace was to make it clear that key bindings != keys.
I understand that. But then I looked into `InputUtil` and saw that:
```Java
mapKey(KEYSYM, "key.keyboard.0", GLFW.GLFW_KEY_0);
```
And I though it would be better to have the same format

Reply:
Well, the vanilla keys are sadly not namespaced at all.

The namespacing for the key bindings is taken from how Fabric API used to enforce key binding ids. I thought it'd make sense sticking with that for the keybindings but making it clear that keys are something different.

Reply:
Okay. I am a bit confused now. Leaving this for the talk